src/redux/store.js

```diff
@@ -1,7 +1,27 @@
 import { createStore, combineReducers } from 'redux';
+import AsyncStorage from '@react-native-async-storage/async-storage'
+import { persistStore, persistReducer } from 'redux-persist';
 
-//import xx from './reducers';
+import countReducer from './reducers';
 
-const rootReducer = combineReducers({});
+const rootReducer = combineReducers({ countReducer: countReducer });
 
-export const store = createStore(rootReducer);
\ No newline at end of file
+//Redux Persist Config
+const persistConfig = {
+    key: 'root',
+    storage: AsyncStorage,
+    whitelist: [
+        'countReducer',
+    ],
+};
```

Comment:
It'd probably be useful to add a `version` option here as well. Now that the state is persisted to local storage, it's possible we'll load a state made with an older version of the app. Having a version allows us to specify ways to migrate older states to newer states: https://github.com/rt2zz/redux-persist/blob/master/docs/migrations.md